src/wallet.js

```diff
@@ -258,13 +261,32 @@ const wallet = {
    */
   storeEncryptedWords(words, password) {
     const initialAccessData = this.getWalletAccessData() || {};
-
     const encryptedDataWords = this.encryptData(words, password);
+
     initialAccessData['words'] = encryptedDataWords.encrypted.toString();
 
     this.setWalletAccessData(initialAccessData);
   },
 
+  /**
+   * Encrypt a auth xpriv to be used for auth on the Wallet Service facade with
+   * a password and save it on localStorage
+   *
+   * @param {string} key Key to be encrypted
+   * @param {string} password Password to encrypt
+   *
+   * @memberof Wallet
+   * @inner
+   */
+  storeEncryptedAuthKey(key, password) {
```

Comment:
It would make sense as it is exclusively being used by the wallet service facade

But I prefer keeping it on `src/wallet.js` so everything that is related to data storage is on this file, we don't have to find what is mutating the storage data anywhere else on future code changes

What do you think?